test/integration/etcd/etcd_storage_path_test.go

```diff
@@ -531,10 +533,14 @@ func startRealMasterOrDie(t *testing.T) (*allClient, clientv3.KV, meta.RESTMappe
 	kubeClientConfigValue := atomic.Value{}
 	storageConfigValue := atomic.Value{}
 
+	certDir, _ := ioutil.TempDir("", "test-integration-etcd")
+	defer os.RemoveAll(certDir)
```

Comment:
is this the right place to put this? I'm guessing the api server keeps running after this function returns?